push_tasks.rb

```diff
@@ -0,0 +1,28 @@
+dep 'schema up to date.push', :ref, :remote, :env do
+  def db_name
+    config = YAML.load_file('config/database.yml'.p)[env.to_s]
+    config['database'] || config['url']
```

Comment:
Fall back through `database` and `url` properties. The `pg_dump` command can handle both variants.